index.html

```diff
@@ -59,71 +56,144 @@ <h1 class="header-title">
+              alt="description-section__img"
+            />
+          </div>
+          <div
+            class="description-section__column description-section__column-info"
+          >
+            <div>
+              <span class="description-section__benefit-text"
+                >Create an account</span
+              >
+              <h2 class="description-section__title">
+                Create/login to an existing<br />
+                account to get started
+              </h2>
+              <p class="description-section__description">
+                An account is created with your email<br />
+                and a desired password
+              </p>
             </div>
+          </div>
         </div>
       </div>
     </section>
-    <section class="interface-app">
-       <div class="container">
-         <div class="interface-app__description">
-           <div class="interface-app__item">
-             <h3 class="interface-app__explore">Explore varieties</h3>
-             <h1 class="interface-app__advantages">Shop for your favorites<br> as e dey hot.</h1>
-             <h4 class="interface-app__description">Shop for your favorite meals or drinks<br> and enjoy while doing it.</h4>
+
+    <section class="description-section">
+      <div class="sm-container">
+        <div class="description-section__columns reverse">
+          <div
+            class="description-section__column description-section__column-img"
+          >
+            <img
+              src="images/Rectangle (2).png"
+              alt="description-section__img"
+            />
+          </div>
+          <div
+            class="description-section__column description-section__column-info"
+          >
+            <div>
+              <span class="description-section__benefit-text"
+                >Explore varieties</span
+              >
+              <h2 class="description-section__title">Shop for your favorites <br>meal as e dey hot.</h2>
+              <p class="description-section__description">Shop for your favorite meals or drinks <br>and enjoy while doing it.</p>
             </div>
-            <img class="interface-app__img" src="images/Rectangle (2).png" alt="interface-app__img">
           </div>
+        </div>
       </div>
     </section>
-    <section class="payment-info">
-      <div class="container">
-        <div class="payment-info__description">
-          <img class="payment-info__img" src="images/Rectangle 3.png" alt="payment-info__img">
-            <div class="payment-info__item">
-              <h3 class="payment-info__checkout">Checkout</h3>
-              <h1 class="payment-info__advantages">When you done check out<br> and get it delivered.</h1>
-              <h4 class="payment-info__deliver">When you done check out and get it<br> delivered with ease.</h4>
+    <section class="description-section">
+      <div class="sm-container">
+        <div class="description-section__columns">
```

Comment:
let's use description-section__row so we will be consistent with bootstrap in future